### DISCUSSION TITLE
Block editor: editing without it?

### DISCUSSION THREAD

**Discussion by slowscape** [automadcms/discussions#17] [General]
Automad 1.10.9

Sometimes while editing content I will see some glitchy things happen. I don't know if this is an editor issue, or if it's an issue of things not saving properly when I hit publish, or something else. 

I was wondering if there was a way to see / edit the raw block editor data? I feel like there was, but I could be thinking of another platform. It's been a year or so since I initially set up Automad. 

Thank you!

**Comment by marcantondahmen** [comment]
Hi, it's a flat-file CMS. All content is stored in files that can be edited with a text editor. 

I recommend using version 2. It has a new editor. Also all content is now saved in JSON format.

**Reply by slowscape** [reply]
I knew I could access the flat file but was wondering if there was a dashboard way, I must have been thinking of some other platform where I could access the raw text.

Either way, I took your advice and have been setting up v2. So far it feels a lot more sturdy, I'm liking it! 
Quick question about structure: I want to create a blog as part of my otherwise non-blog website. 

I created a blog page where the blog posts (pages) will list. I then create the pages under the blog, rendering the page slug `/blog/blog-post` 
Is there a non htaccess way (something built in to automad 2) of removing `/blog/` from the blog post's slug?

**Reply by marcantondahmen** [reply]
Not really, the routing is based on the directory structure. You can add aliases for each post but I guess that is not really practical for a larger amount of pages.
